pkg/providers/instancetype/instancetypes.go

```diff
@@ -46,6 +46,9 @@ import (
 const (
 	InstanceTypesCacheKey = "types"
 	InstanceTypesCacheTTL = 23 * time.Hour
+
+	Ubuntu2204ImageFamily = "Ubuntu2204"
+	AzureLinuxImageFamily = "AzureLinux"
```

Comment:
Yes, we have them in ImageFamily but we get a circular dependency IIRC? Could be wrong its been a couple of weeks since I looked at this, will probably remove these anyway since I will be moving filtering by image family into the instance provider

Reply:
Moved them to the v1alpha2 api, to follow the pattern AWS provider takes.